sdk/keyvault/azure-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/models/Secret.java

```diff
@@ -22,13 +22,13 @@
      * The value of the secret.
      */
     @JsonProperty(value = "value")
-    private String value;
+    private final String value;
```

Comment:
Will this break any serialization by adding final here?

Reply:
yeah, this is not right.
It looks I did this change before updating the rule to ignore fields annotated with `JsonProperty`

Reply:
@conniey , I found this while looking if there was any other JsonProperty that I might updated
```
public class RecordedData {
    @JsonProperty()
    private final LinkedList<NetworkCallRecord> networkCallRecords;
    @JsonProperty()
    private final LinkedList<String> variables;
```

but this was not part of my changes..  should we remove the `final` from there?  or this is not in use?

Reply:
If it's working, I'd rather not remove it. It is assigned once in the default constructor. And maybe Jackson serializer is smart enough to check if there's the variable is not null, and if not just adds contents into that list.